packages/cli/src/cmds/validator/signers/index.ts

```diff
@@ -39,7 +39,12 @@ export async function getSignersFromArgs(args: IValidatorCliArgs & IGlobalArgs,
   // ONLY USE FOR TESTNETS - Derive interop keys
   if (args.interopIndexes) {
     const indexes = parseRange(args.interopIndexes);
-    return indexes.map((index) => ({type: SignerType.Local, secretKey: interopSecretKey(index)}));
+    // Using a remote signer with TESTNETS
+    if (args["externalSigner.pubkeys"] || args["externalSigner.fetch"]) {
+      return getRemoteSigners(args);
+    } else {
+      return indexes.map((index) => ({type: SignerType.Local, secretKey: interopSecretKey(index)}));
+    }
```

Comment:
What's the purpose of this? I don't follow

Reply:
To consume the interop indexes from the web3signer you should  load the interop keystores in the web3signer and then use .fetch